LoLTournament/Models/StatisticsViewModel.cs

```diff
@@ -26,12 +27,14 @@ public StatisticsViewModel()
                 AvgDeaths = matches.Sum(x => x.DeathsBlueTeam + x.DeathsPurpleTeam) / TotalGames;
                 AvgAssists = matches.Sum(x => x.AssistsBlueTeam + x.AssistsPurpleTeam) / TotalGames;
                 AvgMatchDuration = TimeSpan.FromSeconds(matches.Average(x => x.Duration.TotalSeconds));
+                BlueSideWinPercentage = matches.Where(match => match.Finished).Where(match => match.BlueTeamId == match.WinnerId).Count() / TotalGames;
```

Comment:
Cast `TotalGames` to `double` first maybe? Or does `Count()` return a `double` already?